docs/notebooks/Training_with_custom_hyperparameters.ipynb

```diff
@@ -300,11 +300,11 @@
         "id": "KQhv_gsdJzaq"
       },
       "source": [
-        "I'll assume you already know how to train using a training package on Colab. If you haven't done this already, you should take a look at our [other notebook](https://colab.research.google.com/github/murthylab/sleap/blob/main/docs/notebooks/Training_and_inference_using_Google_Drive.ipynb) which walks you through that process in more detail.\n",
+        "We'll assume you already know how to train using a training job package on Colab. If you haven't done this already, you should take a look at our [other notebook](https://colab.research.google.com/github/murthylab/sleap/blob/main/docs/notebooks/Training_and_inference_using_Google_Drive.ipynb) which walks you through that process in more detail.\n",
```

Comment:
We don't want to just link to the rendered page on sleap.ai?

Reply:
Done